config/environments/test.rb

```diff
@@ -39,4 +39,12 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+  config.paperclip_defaults = {
+    storage: :s3,
+    s3_credentials: {
+      bucket: "cardapp",
+      access_key_id: ENV['s3_key'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.